packages/flutter/test/widgets/form_test.dart

```diff
@@ -864,6 +865,7 @@ void main() {
                 key: formKey,
                 child: TextFormField(
                   maxLength: 5,
+                  maxLengthEnforcement: MaxLengthEnforcement.allowComposingTextToFinish,
```

Comment:
Was this test added in the previous PR? The issue number doesn't seem to be right...

Reply:
#63753